pkg/api/customization/cluster/formatter.go

```diff
@@ -47,22 +47,32 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 			resource.AddAction(request, v3.ClusterActionBackupEtcd)
 			resource.AddAction(request, v3.ClusterActionRestoreFromEtcdBackup)
 		}
-		canUpdateClusterFn := func(request *types.APIContext, clusterID string) bool {
-			cluster := map[string]interface{}{
-				"id": clusterID,
-			}
-			return request.AccessControl.CanDo(
-				v3.ClusterGroupVersionKind.Group,
-				v3.ClusterResource.Name,
-				"update",
-				request,
-				cluster,
-				resource.Schema) == nil
+		isActiveCluster := false
+		if resource.Values["state"] == "active" {
+			isActiveCluster = true
+		}
+		isWindowsCluster := false
+		if resource.Values["windowsPreferedCluster"] == true {
+			isWindowsCluster = true
 		}
-		canUpdateCluster := canUpdateClusterFn(request, resource.ID)
-		logrus.Debugf("user: %v, canUpdateCluster: %v", request.Request.Header.Get("Impersonate-User"), canUpdateCluster)
-		if canUpdateCluster {
-			resource.AddAction(request, v3.ClusterActionRunSecurityScan)
+		if isActiveCluster && !isWindowsCluster {
+			canUpdateClusterFn := func(request *types.APIContext, clusterID string) bool {
```

Comment:
I realize this is just being carried over from before but the func isn't being used again and makes the code harder to follow vs just building the structs and making the access call.

Reply:
moved it out to a separate function.